frameworks.md

```diff
@@ -119,26 +127,98 @@ Execution flow:
   * test  *bb*  starts
   * test  *bb*  ends
   * module  *a > b*  ends
-  * module  *a > c > ca*  starts
-  * test  *cc*  starts
-  * test  *cc*  ends
-  * module  *a > c > ca*  ends
+  * module  *a > c > ca* starts
+  * test *cca* starts
+  * test *cca* ends
+  * module *a > c > ca* ends
+  * module *a > c* starts
+  * test *cc* starts
+  * test *cc* starts
+  * module *a > c* ends
   * module  *a*  starts
   * test  *aa*  starts
   * test  *aa*  ends
   * module  *a*  ends
 
-As you can see neither the module *a > c* is not emitted for its start and end, because it does not contain a test itself.  
+Reporter output:
+* a: aa (1)
+* a > b: bb (1)
+* a > c: cc (1)
+* a > c > ca: cca (1)
 
 The *QUnit.config.modules* will contain 5 modules:
   0. module *a*
   1. module *a > b*
   2. module *a > c*
   3. module *a > c > a*
   4. module *a > d*
- 
+
 **The above execution flow is the default one**, QUnit has also 2 options that randomizes tests execution:
-  1. the [reorder](http://api.qunitjs.com/QUnit.config/) option that on a rerun, runs firstly the failed tests, it is activated by default 
+  1. the [reorder](http://api.qunitjs.com/QUnit.config/) option that on a rerun, runs firstly the failed tests, it is activated by default
   2. the [seed](http://api.qunitjs.com/QUnit.config/) option that randomizes tests execution, it is disabled by default
-  
+
 **The QUnit.config.modules will always contain the suites in the same order!**
+
+## Jasmine
+[Jasmine](http://jasmine.github.io/) is another testing framework with builtin assertions, but tests will pass even without containg at least one.
```

Comment:
s/containg/containing

Consider making the `, but tests...` a separate sentence, currently its pretty confusing. "at least one" what?